src/modules/mc_pos_control/MulticopterPositionControl.hpp

```diff
@@ -138,16 +138,17 @@ class MulticopterPositionControl : public ModuleBase<MulticopterPositionControl>
 		(ParamFloat<px4::params::MPC_THR_MAX>) _param_mpc_thr_max
 	);
 
-	control::BlockDerivative _vel_x_deriv; /**< velocity derivative in x */
-	control::BlockDerivative _vel_y_deriv; /**< velocity derivative in y */
-	control::BlockDerivative _vel_z_deriv; /**< velocity derivative in z */
+	static constexpr uint32_t SCHEDULE_INTERVAL_US{20_ms};
+	static constexpr uint32_t SCHEDULE_RATE_HZ{1_s / SCHEDULE_INTERVAL_US};
+
+	matrix::Vector3f _velocity{};
```

Comment:
```suggestion
	matrix::Vector3f _velocity_last{};
```
to make it extra clear?